app/services/create_domain.rb

```diff
@@ -41,7 +41,10 @@ def payload
         name: (0...12).map { (65 + rand(26)).chr }.join + '.ee',
         registrant: ENV['ACCR_CONTACT_CODE'],
         period: 1,
-        period_unit: 'y'
+        period_unit: 'y',
+        'admin_contacts': [
```

Comment:
Should single quotes be used on the key here, if all other keys are 'quoteless'?

Reply:
Yes, because the registry expects to receive a specific string character. Ruby strings are denoted by either single quotes or double quotes. They work the same